install/index.md

```diff
@@ -304,10 +318,26 @@ Windows users can now tap into GPU accelerated data science on their local machi
 
 <i class="fas fa-info-circle text-white"></i> When installing with Docker Desktop, if the container pull command is successful, but the run command hangs indefinitely, [ensure you're on Docker Desktop >= 4.18](https://docs.docker.com/desktop/release-notes/){: target="_blank"}.
 
+<br/>
+<div id="wsl2-sdkm"></div>
+
+### **WSL2 SDK Manager Install (Preferred Method)**
```

Comment:
Move this below conda, Docker, and pip. We should not claim that this is a "preferred" method.
```suggestion
### **WSL2 SDK Manager Install**
```

Reply:
i'll keep it above, but will remove preferred